extensions/iobj-lib/standalone.mjs

```diff
@@ -3,11 +3,16 @@
 
 import assert from 'assert';
 import fs from 'fs';
-
 import {randomShuffle, randomChoice} from '../../common/utility/random-tool.mjs';
 import InteractiveObjectServerBaseClass from '../../common/interactive-object/server.mjs';
 import {getRunPath, getConfigPath} from '../../common/path-util/path.mjs';
 
+// Boilerplate for getting require() in es module.
+import {createRequire} from 'module';
+const require = createRequire(import.meta.url);
+
+const otplib = require('otplib');
+
```

Comment:
Is there a way to use `import` instead of `require()`? If yes, then I would suggest not using `require()` to make our code more consistent. (and explicit file path used in `import` is more clear than a mere module name used in `require()` to me)

Reply:
I tried import 'otplib' as shown by the otplib documentations, but somehow yarn doesn't like it.